docs/workflow_solutions/getting_containers.md

```diff
@@ -287,6 +287,66 @@ python python_test.py
+
+### How to Run UAB RC Community Containers
+
+To use a container from the registry, navigate to the preferred container. On the left navigation pane, you will see an option "Deploy", select this and a list of options will show up.
+
+![Accessing a container in a GitLab container registry](./images/gitlab-deploy-container.png)
+
+Click on "Container Registry" and then click the container, usually in the format "`containername/containername`", this would open a new page displaying the tag(s) (usually the `Commit SHA`). Use the copy icon next to the tag to copy the container's image path.
+
+![Steps for copying an image path in the GitLab Container Registry](./images/gitlab-container-tag.png)
+
+The next set of instructions will guide you through how to complete the install process for the container on Cheaha or on other platforms.
+
+#### Run UAB RC Community Containers on Cheaha using Singularity
+
+On Cheaha, you can pull a container using Singularity by running the `singularity pull` command, an example is shown below. Ensure you are in the preferred location you would like to use the container from.
+
+``` bash
+singularity pull <preferredName.sif> docker://gitlab.rc.uab.edu:<0000>/rc-data-science/community-containers/<name of container/name of container:tag>
+```
+
+In the above command, `<preferredName.sif>` is a file name placeholder you should replace with your preferred container image filename. For example you can rename the file as "`alphafold3.sif`". The `<0000>` is a placeholder for the specific port for our GitLab container registry (at this time it is 4567). For the `<name of container/name of container:tag>` placeholder, you will need to replace this placeholder with the specific path with the image name and tag of the container you want to pull.
+
+A sample code is shown below
+
+``` bash
+singularity pull alphafold3.sif docker://gitlab.rc.uab.edu:4567/rc-data-science/community-containers/alphafold3/alphafold3:cd48cee5
+```
+
+After the singularity file (.sif) is created, you can run your container using the `singularity run` or `singularity exec` commands with the appropriate options. Please refer to the `README` file provided in the container repository or look for documentation specific to the container. We have a guide for using Containers (with singularity) on Cheaha, you can find it [here](#containers-on-cheaha).You should see an output like in the image below.
+
+![Output after running singularity pull command](./images/singularity-terminal-ouput.png)
+
+#### Run UAB RC Community Containers on a Virtual Machine (e.g. cloud.rc) or on Windows using Docker
+
+You can also pull this container using Docker assuming you are running a Virtual Machine (VM) on a cloud instance or on a Windows machine. You will need to have Docker installed, then pull a container from the registry by running the command in your machine's terminal.
```

Comment:
Make it clear that Docker is an alternative to singularity. They are free to install either on a VM.